Source/LinqToDB.EntityFrameworkCore/ILinqToDBForEFTools.cs

```diff
@@ -14,6 +14,8 @@ namespace LinqToDB.EntityFrameworkCore
 	using Mapping;
 	using Metadata;
 	using Data;
+	using System.Collections.Generic;
+	using LinqToDB.Interceptors;

```

Comment:
`System.Collections.Generic` should go at the top with the other `System` usings; `using Interceptors;` since `LinqToDB` is referenced by the namespace (to match the other l2db usings). Check all updated files similarly.

Reply:
Adjusted.